stdlib/universe/universe.flux

```diff
@@ -291,6 +292,13 @@ truncateTimeColumn = (timeColumn="_time", unit, tables=<-) =>
     tables
         |> map(fn:(r) => ({r with _time: date.truncate(t: r._time, unit: unit)}))
 
+// AggregateMonths applies an aggregate function to calendar months.
+aggregateMonths = (fn, column="_value", tables=<-) =>
+    tables
+        |> map(fn: (r) => ({r with monthYear: strings.joinStr(arr: [strings.substring(v: string(v: date.year(t: r._time)*100 + date.month(t: r._time)), start: 0, end: 4), strings.substring(v: string(v: date.year(t: r._time)*100 + date.month(t: r._time)), start: 4, end: 6)], v: "-")}))
```

Comment:
I was thinking about this later....what if we made `monthYear` type time instead of type string?  it should still have the same outcome.

also makes me think of `windowMonth`  that sets the _start as midnight on the first of that month, and _stop as 11:59 on the last day of that month.